src/openapi.yaml

```diff
@@ -1232,22 +1224,22 @@ components:
         success:
           type: string
           format: html
-          description: wip
+          description: wip descriptive note detailng success
```

Comment:
confirm pls

Reply:
Describes a successful outcome of the protocol.